crates/beerus-rpc/src/api.rs

```diff
@@ -16,6 +17,10 @@ use starknet::{
         Transaction,
     },
 };
+use std::string::ToString;
+pub const BLOCK_NOT_FOUND: &str = "Block not found";
+pub const CONTRACT_NOT_FOUND: &str = "Contract not found";
+pub const CONTRACT_ERROR: &str = "Contract error";
```

Comment:
If we go this way I think it would be ideal to separate these constants in another file.

Reply:
good for now, can separate at a later date